align_system/cli/run_align_system.py

```diff
@@ -144,10 +150,10 @@ def _compute_time_stats(times_s):
         else:
             log.info(alignment_target)
             if save_alignment_targets_to_path is not None:
-                alignment_target_path = os.path.join(save_alignment_targets_to_path, f"{alignment_target.id}.yaml")
+                alignment_target_path = os.path.join(save_alignment_targets_to_path, f"{alignment_target.id}.json")
 
                 with open(alignment_target_path, "w") as f:
-                    yaml.dump(alignment_target.to_dict(), f)
+                    json.dump(alignment_target.to_dict(), f, indent=2)
```

Comment:
@eveenhuis just a heads up that I'm changing this output format to JSON (instead of YAML) to be consistent with the other output files we save from this script (and the above `OmegaConf.to_container` call is necessary to be able to serialize the target to JSON if we explicitly set the target with the Hydra config)

Reply:
Sounds good, we have [this script](https://github.com/ITM-Kitware/scratch/blob/master/get_soartech_alignment_targets.py) to pull down alignment targets if we ever need the yaml format anyway